src/pip/_internal/locations/_distutils.py

```diff
@@ -32,7 +35,13 @@ def _distutils_scheme(
         dist_args["script_args"] = ["--no-user-cfg"]
 
     d = Distribution(dist_args)
-    d.parse_config_files()
+    try:
+        d.parse_config_files()
+    except UnicodeDecodeError:
+        logger.warning(
+            "Can't parse distutils configs. Ignoring [global] options in %s.",
```

Comment:
`[install]` too IIUC.

Reply:
Reviewing this again, I suspect there are some more (not sure, distutils documentation is scarce). I think I’ll rewrite the message to avoid mentioning section names here.

Reply:
"parse" could also mean that one of the files is malformed, e.g. missing a closing bracket somewhere, so I'd rather this mentioned it's a decoding error.

Reply:
Ah, it looks like you had the same thought :)